CSharp/SchemaVersion/packages.config

```diff
@@ -1,21 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net47" />
-  <package id="Microsoft.Data.Sqlite" version="3.1.2" targetFramework="net47" />
-  <package id="Microsoft.Data.Sqlite.Core" version="3.1.2" targetFramework="net47" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.0" targetFramework="net47" />
+  <package id="Microsoft.Data.Sqlite" version="3.1.3" targetFramework="net47" />
+  <package id="Microsoft.Data.Sqlite.Core" version="3.1.3" targetFramework="net47" />
   <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net47" />
   <package id="Npgsql" version="4.1.3.1" targetFramework="net47" />
   <package id="NuoDb.Data.Client" version="2.3.0.12" targetFramework="net47" />
   <package id="SQLitePCLRaw.bundle_e_sqlite3" version="2.0.2" targetFramework="net47" />
   <package id="SQLitePCLRaw.core" version="2.0.2" targetFramework="net47" />
   <package id="SQLitePCLRaw.lib.e_sqlite3" version="2.0.2" targetFramework="net47" />
   <package id="SQLitePCLRaw.provider.dynamic_cdecl" version="2.0.2" targetFramework="net47" />
-  <package id="System.Buffers" version="4.5.0" targetFramework="net47" />
-  <package id="System.Memory" version="4.5.3" targetFramework="net47" />
+  <package id="System.Buffers" version="4.5.1" targetFramework="net47" />
+  <package id="System.Memory" version="4.5.4" targetFramework="net47" />
   <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net47" />
-  <package id="System.Runtime.CompilerServices.Unsafe" version="4.6.0" targetFramework="net47" />
-  <package id="System.Text.Encodings.Web" version="4.6.0" targetFramework="net47" />
-  <package id="System.Text.Json" version="4.6.0" targetFramework="net47" />
-  <package id="System.Threading.Tasks.Extensions" version="4.5.3" targetFramework="net47" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.7.1" targetFramework="net47" />
+  <package id="System.Text.Encodings.Web" version="4.7.0" targetFramework="net47" />
+  <package id="System.Text.Json" version="4.7.1" targetFramework="net47" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.4" targetFramework="net47" />
   <package id="System.ValueTuple" version="4.5.0" targetFramework="net47" />
 </packages>
```

Comment:
Carriage return

Reply:
@JacobPenland  Thanks for pointing those out, you know how much I dislike them :-)
I will make the edits in the develop branch.